db/fixtures/miq_product_features.yml

```diff
@@ -696,26 +696,18 @@
         :description: Remove Catalog Items from the VMDB
         :feature_type: admin
         :identifier: catalogitem_delete
-      - :name: Edit Composite Catalog Item
-        :description: Edit a Composite Catalog Item
+      - :name: Edit Atomic/Composite Catalog Item
+        :description: Edit an/a Atomic/Composite Catalog Item
```

Comment:
`Edit an/a Atomic/Composite Catalog Item` can just be `Edit a Catalog Item`.  Same in all of these features.